packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -211,6 +211,7 @@
   @include u-square($size-touch-target);
   background-color: color("black-transparent-10");
   display: inline-block;
+  padding: units(0.5);
```

Comment:
Discovered the social icons were missing the padding they have currently in `develop`. If this was left out intentionally, please let me know.

Reply:
Not intentional! Thank you for adding.